packages/core/import-map.importmap

```diff
@@ -8,6 +8,7 @@
     "lit-html/lib/shady-render.js": "/base/node_modules/lit-html/lib/shady-render.js",
     "lit-html/directives/if-defined.js": "/base/node_modules/lit-html/directives/if-defined.js",
     "ramda/es/anyPass.js": "/base/node_modules/ramda/es/anyPass.js",
+    "ramda/es/clone.js": "/base/node_modules/ramda/es/clone.js",
```

Comment:
We use clone to make copies of animation objects so we don't pass around references and unintentionally mutate animation configs.